ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -28,18 +28,48 @@
           (servlet/update-servlet-response response response-map)
           (.setHandled base-request true))))))
 
-(defn- ^AbstractHandler async-proxy-handler [handler timeout]
+(defn- async-jetty-raise
```

Comment:
This could be more concisely written as:

```clojure
(defn- async-jetty-raise [^AsyncContext context ^HttpServletResponse response]
  (fn [^Throwable exception]
    (.sendError response 500 (.getMessage exception))
    (.complete context)))
```